packages/calcite-components/.eslintrc.cjs

```diff
@@ -73,6 +73,10 @@ module.exports = {
     "import/no-dynamic-require": ["error", { esmodule: true }],
     "import/order": ["error", { "newlines-between": "never" }],
     "jest/no-export": "warn",
+    /* Using conditional logic in a confined test helper to handle specific scenarios, reducing duplication, balancing test readability and maintainability. **/
```

Comment:
I suggested adding a new `.eslintrc.cjs` under `commonTests`. The root ESLint configuration should not be modified.

Reply:
Got it.